src/randoop/operation/ConstructorCall.java

```diff
@@ -7,51 +7,39 @@
 import java.lang.reflect.Modifier;
 import java.util.ArrayList;
 import java.util.Arrays;
-import java.util.HashSet;
 import java.util.List;
-import java.util.Set;
 
 import randoop.ExceptionalExecution;
 import randoop.ExecutionOutcome;
-import randoop.Globals;
 import randoop.NormalExecution;
 import randoop.main.GenInputsAbstract;
-import randoop.sequence.ExecutableSequence;
+import randoop.reflection.ReflectionPredicate;
+import randoop.sequence.Statement;
 import randoop.sequence.Variable;
+import randoop.types.TypeNames;
 import randoop.util.ConstructorReflectionCode;
-import randoop.util.PrimitiveTypes;
-import randoop.util.Reflection;
 import randoop.util.ReflectionExecutor;
 import randoop.util.Util;
 
 /**
- * Represents a constructor call. The inputs are parameters to the constructor
- * and outputs include the new object.
- *
- * The "R" stands for "Randoop", to underline the distinction from
- * java.lang.reflect.Constructor.
- *
+ * ConstructorCall is an {@link Operation} that represents a call to a constructor, and holds a reference to
+ * a reflective {@link java.lang.reflect.Constructor} object.  
+ * 
+ * As an {@link Operation}, a call to constructor c with n arguments is represented as 
+ * c : [t1,...,tn] -> c, where the output type c is also the name of the class. 
```

Comment:
c appears twice at the beginning of this line.  Is the first one representing "new C"?